srcpkgs/oomox/template

```diff
@@ -15,28 +17,33 @@ maintainer="Tim Wanders <timwanders241@gmail.com>"
 license="GPL-3"
 homepage="https://github.com/themix-project/oomox"
 distfiles="
- https://github.com/themix-project/oomox/archive/$version.tar.gz>oomox-$version.tar.gz
- https://github.com/themix-project/oomox-archdroid-icon-theme/archive/$_droid_icon_ver.tar.gz>oomox-archdroid-icon-theme-$_droid_icon_ver.tar.gz
- https://github.com/themix-project/oomox-gnome-colors-icon-theme/archive/$_gnome_icon_ver.tar.gz>oomox-gnome-colors-$_gnome_icon_ver.tar.gz
- https://github.com/themix-project/oomoxify/archive/$_oomoxify_ver.tar.gz>oomoxify-$_oomoxify_ver.tar.gz
- https://github.com/nana-4/materia-theme/archive/$_materia_theme_ver.tar.gz>materia-theme-$_materia_theme_ver.tar.gz
- https://github.com/themix-project/oomox-gtk-theme/archive/$_oomox_gtk_theme_ver.tar.gz>oomox-gtk-theme-$_oomox_gtk_theme_ver.tar.gz"
+	https://github.com/themix-project/oomox/archive/$version.tar.gz>oomox-$version.tar.gz
+	https://github.com/themix-project/oomox-archdroid-icon-theme/archive/$_droid_icon_ver.tar.gz>oomox-archdroid-icon-theme-$_droid_icon_ver.tar.gz
+	https://github.com/themix-project/oomox-gnome-colors-icon-theme/archive/$_gnome_icon_ver.tar.gz>oomox-gnome-colors-$_gnome_icon_ver.tar.gz
+	https://github.com/themix-project/oomoxify/archive/$_oomoxify_ver.tar.gz>oomoxify-$_oomoxify_ver.tar.gz
+	https://github.com/nana-4/materia-theme/archive/v$_materia_theme_ver.tar.gz>materia-theme-$_materia_theme_ver.tar.gz
+	https://github.com/themix-project/oomox-gtk-theme/archive/$_oomox_gtk_theme_ver.tar.gz>oomox-gtk-theme-$_oomox_gtk_theme_ver.tar.gz
+	https://github.com/actionless/arc-theme/archive/$_arc_theme_ver.tar.gz>arc-theme-$_arc_theme_ver.tar.gz
```

Comment:
here you can already use upstream repo: https://github.com/NicoHood/arc-theme

but instead of tagged version you have to use some latest commit